datafusion/sql/src/unparser/expr.rs

```diff
@@ -561,6 +636,31 @@ mod tests {
                 }),
                 r#"CAST("a" AS INTEGER UNSIGNED)"#,
             ),
+            (
+                Expr::InList(InList {
```

Comment:
I think you can write this more concisely with the expr API -- https://docs.rs/datafusion/latest/datafusion/logical_expr/enum.Expr.html#method.in_list

Something like
```rust
col("a").in(vec![lit(1), lit(2), lit(3)])
```

Likewise for NOT IN

```rust
col("a").not_in(vec![lit(1), lit(2), lit(3)])
```